geoprocessing/src/modules/protected-areas/protected-areas.geo.entity.ts

```diff
@@ -2,36 +2,94 @@
  * @todo We are replicating the same code that we have in the api. If we update something here we should also replicate it in the api side.
  */
 import { ApiProperty } from '@nestjs/swagger';
-import { Column, Entity, PrimaryColumn } from 'typeorm';
-import { Geometry } from 'geojson';
+import { Check, Column, Entity, Index, PrimaryGeneratedColumn } from 'typeorm';
+import { MultiPolygon } from 'geojson';
+import { TimeUserEntityMetadata } from '../../types/time-user-entity-metadata';
+
+/**
+ *  represents spatial data using longitude and latitude coordinates on the Earth's surface as defined in the WGS84 standard, which is also used for the Global Positioning System (GPS)
+ */
+const srid = 4326;
```

Comment:
I would even go as far as making this a shared constant across the service, as in practice we rarely use anything else than this srid... maybe calling it `defaultSrid` or even `srid4326` (but this whole thing would make more sense if enforcing via static analysis that we don't use the raw 4326 identified anywhere and we always use the constant). Anyway, very minor 😄

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/201/commits/2638dea6fc138a4191949944207dc12caa046228